test/use-checkout-pricing.test.js

```diff
@@ -222,19 +210,6 @@ describe('useCheckoutPricing', function() {
   describe('Error handler', () => {
     it('should be passed to .catch', async () => {
       const handleError = jest.fn();
-      window.Promise.all = function() {
```

Comment:
This is completely unnecessary, I'm not sure why I tried mocking this out in the first place.

I think it had to do with having difficulty with getting this test to pass. For some reason, the `waitForNextUpdate` utility wasn't waiting long enough before checkoutPricing.catch was being called, hence the usage of `setTimeout`.

I'd love to remove the setTimeout in favor of something more consistent, but wasn't able to find a way to do this.